### PR TITLE
Generate project source code

### PR DESCRIPTION
**Pull request by nfqphuong**
#### What does this pr do?
- Generate the project architecture using Symfony binary command

#### Screenshots
image

### CODE REVIEW

File: src/Controller/LuckyController.php [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      */
+    public function number(): Response
+    {
+        $number = random_int(0, 100);
+
+        return $this->render('lucky/number.html.twig', [
+            'number' => $number,
+        ]);
+    }
+}
```
Comment (dinhnhatbang) [inline src/Controller/LuckyController.php]: End of line

**Review by dinhnhatbang — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+/vendor/
+###< symfony/framework-bundle ###
+
+###> symfony/phpunit-bridge ###
+.phpunit
+.phpunit.result.cache
+/phpunit.xml
+###< symfony/phpunit-bridge ###
+
+.idea/
```
Comment (dinhnhatbang) [inline .gitignore]: EOL

File: src/Controller/BlogController.php [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    }
+
+    /**
+     * @Route("/blog/{page<\d+>?1}", name="blog_list", priority="2")
+     */
+    public function list(int $page)
+    {
+        // ...
+    }
+}
```
Comment (dinhnhatbang) [inline src/Controller/BlogController.php]: EOL

File: src/Controller/LuckyController.php (hunk 1 above)
Comment (nfqphuong) [reply src/Controller/LuckyController.php]: I've updated the commit and fixed this issue, please check out the new commit content.

File: .gitignore (hunk 2 above)
Comment (nfqphuong) [reply .gitignore]: I've updated the commit and fixed this issue, please check out the new commit content.

File: src/Controller/BlogController.php (hunk 3 above)
Comment (nfqphuong) [reply src/Controller/BlogController.php]: I've updated the commit and fixed this issue, please check out the new commit content.

**Review by dinhnhatbang — APPROVED** [review]

**Pull request merged**